ApollosProject/apollos-apps
# feat: People Search

## DESCRIPTION

### What does this PR do, or why is it needed?

People search! Using the basis of Jimmy and Luke's excellent work. 

A few notes about this PR: 
- Pagination is un-implemented on the front end. I think that's the right choice for now. 
- Still some UI cleanup to be done on the search modal. We can tackle that in another PR. 

![2021-03-18 21 47 39](https://user-images.githubusercontent.com/1637694/111721023-7a36fb80-8835-11eb-9371-732fb4e8866e.gif)


### How do I test this PR?

Visit the connect tab while linked and hit that shiny "Find People to People" button. Probs should fix my typo 🙃
